akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/breakout/CreateBreakoutRoomsCmdMsgHdlr.scala

```diff
@@ -38,7 +38,7 @@ trait CreateBreakoutRoomsCmdMsgHdlr extends RightsManagementTrait {
     val presId = getPresentationId(state)
     val presSlide = getPresentationSlide(state)
     val parentId = liveMeeting.props.meetingProp.intId
-    var rooms = new collection.immutable.HashMap[String, BreakoutRoom2x]
+    var rooms = new collection.immutable.HashMap[String, BreakoutRoom2x].take(props.meetingProp.maxNumberOfBreakouts)
```

Comment:
I'm not sure what the point of this is. Isn't it going to take N elements from an empty HashMap? To enforce the number of rooms it should be checking the number of rooms passed in msg.body.rooms and rejecting if there's too many.